UI/Session-1/session1.html

```diff
@@ -0,0 +1,209 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Home</title>
+    </head>
+    
+    <body>
+        <header class="header">
+            <div id="logo">
+                <img src="Images/logo.png" alt="Logo Title">
+            </div>
+            
+            <div id="menu">
+                <ul>
+                    <li><a href="">Home</a></li>

```

Comment:
Use the `#` inside blank href - `href="#"` to avoid the unnecessary whole page refresh.